docker/Dockerfile.cpu

```diff
@@ -2,8 +2,6 @@ FROM python:3.9 as base
 
 ARG cnlpt_version
 
-RUN --mount=type=cache,target=/root/.cache \
-  pip install cython
```

Comment:
cnlpt-transformers installs this for us - plus, as a general rule, just installing the one package should be everything we need. If not, we should probably fix that upstream.